blockchain/types/account/account_serializer.go

```diff
@@ -38,12 +43,23 @@ type accountJSON struct {
 // NewAccountSerializer creates a new AccountSerializer object with default values.
 // This returned object will be used for decoding.
 func NewAccountSerializer() *AccountSerializer {
-	return &AccountSerializer{}
+	return &AccountSerializer{preserveExtHash: false}
 }
 
 // NewAccountSerializerWithAccount creates a new AccountSerializer object with the given account.
 func NewAccountSerializerWithAccount(a Account) *AccountSerializer {
-	return &AccountSerializer{a.Type(), a}
+	return &AccountSerializer{a.Type(), a, false}
+}
+
+// NewAccountSerializer creates a new AccountSerializer object with default values.
+// This returned object will be used for decoding.
+func NewAccountSerializerExt() *AccountSerializer {
```

Comment:
Not used in this PR, but used in #1871. It will be used to serialize the state objects (i.e. RLP-encoded accounts).
https://github.com/klaytn/klaytn/blob/v1.10.2/blockchain/state/state_object.go#L131